src/lib/raft/raft.h

```diff
@@ -261,6 +273,15 @@ raft_vote_count(const struct raft *raft)
 	return raft->votes[raft->self].count;
 }
 
+/** Time since the latest communication with the leader. */
+static inline double
+raft_leader_idle(const struct raft *raft)
+{
+	if (raft->is_enabled && raft->state != RAFT_STATE_LEADER)
+		return raft_ev_now(raft_loop()) - raft->leader_last_seen;
```

Comment:
It might be better to use `ev_monotonic_now()` instead of just `ev_now()` - much more stable when it comes to timeouts, deadlines, and when you don't need to send this timestamp anywhere.

Reply:
Thanks! Replaced ev_now with ev_monotonic_now everywhere.